content/en/seachat/seachat-manual/05-integrations/04-seachat-wix-integration.md

```diff
@@ -55,15 +55,26 @@ Use the sizing tool to adjust the chat widget to big enough size. In the example
 
 <img width="80%" style="border-radius: 0.4rem" src="/images/seachat-integrations/wix/wix-seachat-integration-step5.png" alt="Adjust chat widget size">
 
-## Step 7: Save and preview
+## Step 7: Fix SeaChat Widget Location
```

Comment:
Fix sounds like repair. did you mean "fixate"?

* Ensure the widget is anchored in place to avoid movement during page scrolling.
* Modify the widget’s settings to maintain its position fixed on the screen as users scroll.
* Secure the widget’s placement so that it remains stationary when the page is scrolled.